src/components/atoms/panel.tsx

```diff
@@ -1,7 +1,14 @@
 import React from "react";
+type panelProps = {
+  children: React.ReactNode;
+  title?: string;
+};
```

Comment:
Create a `types.ts` file in the same folder and move the type there. 
We keep our types separated from our code